src/test/java/io/vlingo/schemata/model/ContextTest.java

```diff
@@ -7,74 +7,47 @@
-    result.until.completes();
-    Assert.assertTrue(result.defined);
-    Assert.assertEquals(1, result.applied.size());
-    Assert.assertEquals(ContextDefined.class, result.applied.get(0).getClass());
-    Assert.assertFalse(result.described);
-    Assert.assertFalse(result.renamed);
-    Assert.assertFalse(result.recategorised);
-  }
-
-  @Test
-  public void testApplyDescribedVlingoSchemata() throws Exception {
-    result.until.completes();
-    Assert.assertTrue(result.defined);
-    Assert.assertFalse(result.renamed);
-    Assert.assertFalse(result.described);
-    Assert.assertFalse(result.recategorised);
-    Assert.assertEquals(1, result.applied.size());
-    Assert.assertEquals(ContextDefined.class, result.applied.get(0).getClass());
-    result.until = TestUntil.happenings(1);
-    entity.describeAs("desc");
-    result.until.completes();
-    Assert.assertEquals(2, result.applied.size());
-    Assert.assertEquals(ContextDescribed.class, result.applied.get(1).getClass());
-
-  }
-
-  @Test
-  public void applyNamespaceChangedVlingoSchemata() throws Exception {
-    result.until.completes();
-    Assert.assertTrue(result.defined);
-    Assert.assertFalse(result.renamed);
-    Assert.assertFalse(result.described);
-    Assert.assertFalse(result.recategorised);
-    Assert.assertEquals(1, result.applied.size());
-    Assert.assertEquals(ContextDefined.class, result.applied.get(0).getClass());
-    result.until = TestUntil.happenings(1);
-    entity.describeAs("description");
-    result.until.completes();
-    Assert.assertEquals(2, result.applied.size());
-    Assert.assertEquals(ContextDescribed.class, result.applied.get(1).getClass());
-    result.until = TestUntil.happenings(2);
-    entity.changeNamespaceTo("newNamespace");
-    result.until.completes();
-    Assert.assertEquals(3, result.applied.size());
-    Assert.assertEquals(ContextRenamed.class, result.applied.get(2).getClass());
-  }
+    private TestWorld world;
+
+    @Before
+    public void setUp() throws Exception {
+        world = TestWorld.start("context-test");
+    }
+
+    @After
+    public void tearDown() {
+        world.terminate();
+    }
+
+    @Test
+    public void testApplyContextVlingoSchemata() throws Exception {
+        final TestActor<Context> contextTestActor =
+                world.actorFor(Definition.has(ContextEntity.class, Definition.parameters(Id.OrganizationId.unique(), Id.UnitId.unique(),
+                        Id.ContextId.unique(), "namespace", "desc")), Context.class);
+        contextTestActor.actor().changeNamespaceTo("newNamespace");
+        contextTestActor.actor().describeAs("newDesc");
+
+        //Assertion for change namespace
+        Assert.assertEquals(Events.ContextRenamed.class, ((ArrayList) contextTestActor.viewTestState().valueOf("applied")).get(1).getClass());
+        Assert.assertEquals(Events.ContextRenamed.with(Id.OrganizationId.unique(), Id.UnitId.unique(),
```

Comment:
How can these be equals values when the ids are unique? Your `equals()` method must be incomplete. If `assertEquals()` on events with unique ids is impossible to implement, please just do like this:

`final ContextRenamed renamed = contextTestActor.viewTestState().valueOf("applied")).get(0);
assertEquals("newName", renamed.name);
...`